uaclient/security.py

```diff
@@ -550,7 +550,7 @@ def fix_security_issue_id(cfg: UAConfig, issue_id: str) -> None:
             usn=usn, installed_packages=installed_packages
         )
         usn_released_pkgs = merge_usn_released_binary_package_versions(usns)
-        print(usn.get_url_header())
+        print(util.wrap_text(usn.get_url_header()))
```

Comment:
Two things:
1. The issue mentioned indenting: Do we want to add a hanging indent here? We can do so with the `subsequent_indent` kwarg.
2. It might make more sense to move this `util.wrap_text` call into the `get_url_header` method rather than here - thoughts?

Reply:
> Two things:
> 
> 1. The issue mentioned indenting: Do we want to add a hanging indent here? We can do so with the `subsequent_indent` kwarg.

I think we'll probably need an indent there, we defintely should toss this example by @mitechie to get his 
thoughts on the look. Anytime we see a wall-o-text, I cringe and wonder if we should really be hiding some of this content with ellipsis as it becomes unfriendly. here's the example of painful output on a unique USN https://pastebin.canonical.com/p/tXF8xmQD9s/

> 2. It might make more sense to move this `util.wrap_text` call into the `get_url_header` method rather than here - thoughts?

Yes let's move that call inside. best to expect formatted output functions to return us well-formatted output.

Reply:
1. Oof I didn't notice before, but textwrap is totally mangling some of the intended newlines in there. There may be a parameter for textwrap to honor existing newlines, or we can just only call textwrap on the chunk of text that we're dynamically generating and know could go long. I'll look into it

2. Agreed! I'll move it

Reply:
> 1. Oof I didn't notice before, but textwrap is totally mangling some of the intended newlines in there. There may be a parameter for textwrap to honor existing newlines, or we can just only call textwrap on the chunk of text that we're dynamically generating and know could go long. I'll look into it

Thanks I notices that we were missing leading and trailing newlines on that chunk of "wrapped" text. Figured I'd get back to you with a more detailed review why, but it seems I've run out of daylight on this review today :(. Will have to look monday.